testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/cdi/injection/InjectionTest.java

```diff
@@ -157,17 +156,6 @@ public void preparePersistenceTest() throws Exception {
         Response response = base.request().post(Entity.text(new String()));
         invocationCounter++;
         response.close();
-
-        // Create an instance and set it as the singleton to use
-        factory = ResteasyProviderFactory.newInstance();
-        ResteasyProviderFactory.setInstance(factory);
-        RegisterBuiltin.register(factory);
```

Comment:
testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/cdi/injection/InjectionTest.java:
  This removal is OK because all classes in this app only do a getMessageBodyReader
  and getMessageBodyWriter.  Nothing in the test is calling any set data methods.

Reply:
OK, wondering why that code was there, though.